src/coreclr/jit/valuenum.cpp

```diff
@@ -4017,66 +4017,66 @@ ValueNum ValueNumStore::EvalCastForConstantArgs(var_types typ, VNFunc func, Valu
             }
             break;
         }
-            {
```

Comment:
There's no code change here; just fixing the indenting by moving this open scope inside the 'case' below